ci/tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export MIX_ENV="test"
```

Comment:
Sort of surprised this is necessary, since I thought it was set by the `mix test` command already.